rc/.eslintrc.json

```diff
@@ -20,38 +20,38 @@
         "no-array-constructor": 2,
         "no-bitwise": 0,
         "no-caller": 2,
-        "no-catch-shadow": 0,
+        "no-catch-shadow": 2,
         "no-comma-dangle": 2,
         "no-cond-assign": 2,
         "no-console": 2,
         "no-constant-condition": 2,
-        "no-continue": 0,
+        "no-continue": 2,
```

Comment:
This is not the place to have opinions on how to write loops.

We don't have a `no-while` or `no-do-loop` statement either.